docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -100,6 +100,15 @@ Some constraints are due to differences in behavior between Docker and Singulari
    character (") may cause the job to fail with the json.decoder.JSONDecodeError unless you have
    Singularity 3.10 or greater or Apptainer 1.1 or greater.
 
+************************
+ Apptainer Known Issues
+************************
+
+   Determined uses setuid for some of the functionalities, for example, launch Tensorboards and
```

Comment:
Starting with Apptainer 1.1.0, it has separate apptainer-setuid package:
The most significant change is that Apptainer no longer installs a setuid-root portion by default. This is now reasonable to do because most operations can be done with only unprivileged user namespaces (see additional changes below). If installing from rpm or debian packages, the setuid portion can be included by installing the apptainer-suid package,

Reply:
Determined itself doesn't use setuid as part of this, it is more about how Apptainer works.
It may have started even earlier (i.e. 1.1.0) so that would be good to say.